src/_data/site.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "FLOE Project",
+  "shortDesc": "Flexible Learning for Open Education",
+  "url": "https://www.floeproject.org",
+  "authorEmail": "schopra@mt.iitr.ac.in",
+  "authorHandle": "@sachin10101998",
+  "authorName": "Sachin Chopra",
```

Comment:
Probably better to use the org info here.

Reply:
Fixed